app/lib/submission_builder/ty2024/states/nj/documents/schedule_nj_hcc.rb

```diff
@@ -0,0 +1,37 @@
+# frozen_string_literal: true
+
+module SubmissionBuilder
+  module Ty2024
+    module States
+      module Nj
+        module Documents
+          class ScheduleNjHcc < SubmissionBuilder::Document
+            include SubmissionBuilder::FormattingMethods
+
+            def schema_file
+              SchemaFileLoader.load_file("us_states", "unpacked", "NJIndividual2024V0.1", "NJCommon", "SchNJHCC.xsd")
+            end
+
+            def document
+
+              build_xml_doc("SchNJHCC") do |xml|
+                xml.HealthCovAllYear 'X'
+              end
+            end
+
+            private
+            
+            def intake
+              @submission.data_source
```

Comment:
This is included now in the `SubmissionBuilder::Document` file that this submission builder inherits from so no need to include it here [PEBBLE]